streaming/vision/coco.py

```diff
@@ -21,14 +21,14 @@ class COCO(Dataset):
             Defaults to ``None``.
         split (str, optional): The dataset split to use, either 'train' or 'val'. Defaults to
             ``None``.
-        shuffle (bool, optional): Whether to shuffle the train samples in this dataset. Defaults to
+        shuffle (bool, optional):Whether to shuffle the samples in this dataset. Defaults to
```

Comment:
missing space

Reply:
Fixed in https://github.com/mosaicml/streaming/pull/42/commits/c023c30b7f55b89dd9f6526480c0c5a272940bba.